graf3d/gl/CMakeLists.txt

```diff
@@ -221,6 +217,19 @@ ROOT_STANDARD_LIBRARY_PACKAGE(RGL
     ${installoptions}
 )
 
+target_include_directories(RGL PRIVATE ${OPENGL_INCLUDE_DIR})
```

Comment:
in next three lines we can make it more compact adding include dirs [item1, item2] in one call of target_include_directories():
```
target_include_directories(<target> [SYSTEM] [BEFORE]
  <INTERFACE|PUBLIC|PRIVATE> [items1...]
  [<INTERFACE|PUBLIC|PRIVATE> [items2...] ...])
```